createrepo_c.spec

```diff
@@ -4,7 +4,7 @@
 
 %global bash_completion %{_datadir}/bash-completion/completions/*
 
-%if 0%{?rhel} && ( 0%{?rhel} <= 7 || 0%{?rhel} >= 9 )
+%if ( 0%{?rhel} && ( 0%{?rhel} <= 7 || 0%{?rhel} >= 9 ) ) || 0%{?fedora} && 0%{?fedora} >= 39
```

Comment:
Uh, this bcond syntax. It's always confusing... No problem there, I just wanted to vent :slightly_smiling_face:

Reply:
Hmm just thinking if it wouldn't be better to put the new subexpression into parentheses like this: `( 0%{?fedora} && 0%{?fedora} >= 39 )`